docs/machine-learning/resources/tasks.md

```diff
@@ -1,42 +1,52 @@
-# Machine Learning Tasks
-After understanding what you are hoping to achieve with your data, you will need to pick the right machine learning task for your situation. Below, you can find an explanation of different machine learning tasks and some common use cases. 
+---
+title: Machine learning tasks
+description: Explore the different machine learning tasks supported in ML.NET.
+ms.date: 05/30/2018
+---
+# Machine learning tasks
+
+When building a machine learning model, you will first need to define what you are hoping to achieve with your data. After, you can pick the right machine learning task for your situation. The following list will help you understand the different machine learning tasks that you can choose from and some common use cases. 
+
+> [!NOTE]
+> ML.NET is currently in Preview. Not all machine learning tasks are currently supported. To submit a request for a certain task, please open an issue in the [dotnet/machinelearning repository](https://github.com/dotnet/machinelearning/issues).
 
 > [!NOTE]
-> ML.NET is currently in Preview. All machine learning tasks are not currently supported. To submit a request for a certain task, please open an issue on our [GitHub repository](https://github.com/dotnet/machinelearning/issues).
+> Currently, ML.NET does not support machine learning tasks with images. Support will be added in future releases. 
+
+## Binary classification
+
+A supervised machine learning task that is used to predict which of two classes (categories) an instance of data belongs to. The input of a classification algorithm is a set of labeled examples, where each label is an integer of either 0 or 1. The output of a binary classification algorithm is a classifier, which you can use to predict the class of new unlabeled instances. Examples of binary classification scenarios include:
 
-## Binary Classification
-A supervised machine learning task that is used to predict which of two classes an instance of data belongs to. The input of a classification algorithm is a set of labeled examples. Each example is represented as a feature vector, and each label is an integer of value of 0 or 1. The output of a binary classification algorithm is a classifier, which can be used to predict the label of new unlabeled instances. Examples of binary classification scenarios include:
+* [Understanding sentiment of Twitter comments](../tutorials/sentiment-analysis.md) as either "positive" or "negative".
+* Diagnosing whether a medical patient has a certain disease or not.
+* Making a decision to send an email to a "spam" folder or not.
```

Comment:
I would say: *...to mark an email as spam or not*, because *send an email* has other main meaning.